packages/backend/test/peripherals/database/setup.ts

```diff
@@ -0,0 +1,22 @@
+import { getConfig } from '../../../src/config'
+import { DatabaseService } from '../../../src/peripherals/database/DatabaseService'
+
+export function setupDatabaseTestSuite() {
+  const config = getConfig('test')
+  const knex = DatabaseService.createKnexInstance(config.databaseUrl)
+  const skip = config.databaseUrl === 'xXTestDatabaseUrlXx'
```

Comment:
Fair point.

![image](https://user-images.githubusercontent.com/15332326/148388541-d9ff3d38-cd62-47f8-a469-01345f8df200.png)

Reply:
Renamed.

If we'd like to improve upon this more, we could consider something like [`pg-mem`](https://github.com/oguimbal/pg-mem)., but since we run these tests on CI anyway, I feel skipping makes sense.